app/code/Magento/Catalog/Model/Product/Image/Cache.php

```diff
@@ -55,7 +55,7 @@ public function __construct(
      *
      * @return array
      */
-    protected function getData()
+    public function getData()
```

Comment:
Generally changing `protected` to `public` isn't very good, it was designed that way on purpose. You may try looking for a better public method or in the worst case extracting logic of this method to a separate class.